apps/explorer/tests/utils/test_utils.py

```diff
@@ -0,0 +1,160 @@
+import pandas
+import yaml
+
+from ...utils import export_pixelsets
+from apps.core.models import PixelSet
+from apps.core.tests import CoreFixturesTestCase
+from apps.core import factories
+
+
+class ExportPixelsSetTestCase(CoreFixturesTestCase):
+
+    def _assert_archive_is_valid(self, zip_archive):
+        assert len(zip_archive.namelist()) == 2
+        assert 'meta.yaml' in zip_archive.namelist()
+        assert 'pixels.csv' in zip_archive.namelist()
```

Comment:
`'meta.yaml'` and `'pixels.csv'` can be defined once as a configuration:

```python
# apps/explorer/utils/export.py
PIXELSET_EXPORT_META_FILENAME = 'meta.yaml'
PIXELSET_EXPORT_PIXELS_FILENAME = 'pixels.csv'

# apps/explorer/utils/__init__.py
from .export import (
    PIXELSET_EXPORT_META_FILENAME, PIXELSET_EXPORT_PIXELS_FILENAME,
    export_pixelsets
)

__all__ = (
    'PIXELSET_EXPORT_META_FILENAME', 'PIXELSET_EXPORT_PIXELS_FILENAME',
    'export_pixelsets'
)
```

WDYT?

Reply:
makes sense